src/main/java/jenkins/bouncycastle/api/BouncyCastlePlugin.java

```diff
@@ -82,6 +84,14 @@ private final File getOptionalLibDirectory() {
             LOG.log(Level.FINE, "plugindir not specified, falling back to $'{'JENKINS_HOME/plugins'}' as {0}", explodedPluginsDir);
         }
         File f =  new File(explodedPluginsDir, pw.getShortName() + "/WEB-INF/optional-lib/");
+        if (!f.isDirectory()) {
+            File here = Which.jarFile(BouncyCastlePlugin.class);
+            File alt = new File(here.getParentFile().getParentFile(), "optional-lib");
```

Comment:
Possibly this logic is good enough to be the sole tactic, deleting the stuff based on `explodedPluginsDir`, but I wanted to start with a conservative patch.